tapir/coop/forms.py

```diff
@@ -257,13 +257,13 @@ class MembershipResignationForm(forms.ModelForm):
             attrs={"rows": 2, "placeholder": _("Please not more than 1000 characters.")}
         ),
     )
-    coop_buys_shares_back = forms.BooleanField(
-        label=_("Coop buys back share(s)"), required=False
-    )
-    willing_to_gift_shares_to_coop = forms.BooleanField(required=False)
     share_owner = ShareOwnerChoiceField()
-    transfering_shares_to = ShareOwnerChoiceField(
-        required=False, label=_("Transfering share(s) to")
+    transferring_shares_to = ShareOwnerChoiceField(
+        required=False,
+        label=_("Transferring share(s) to"),
+        help_text=MembershipResignation._meta.get_field(
```

Comment:
What is the _meta here?

Reply:
It's created automatically by Django on all models, the doc is [here](https://docs.djangoproject.com/en/3.2/ref/models/meta/)

Reply:
Here I use it get the the field definition so that I can reuse the help text instead of copying it.